src/test/java/org/minions/devfund/carlosorellana/SquarelotronTest.java

```diff
@@ -0,0 +1,40 @@
+package org.minions.devfund.carlosorellana;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * The goal is test the Squarelotron methods.
+ *
+ * @author Carlos Orellana.
+ * @since 07/17/2018.
+ */
+public class SquarelotronTest {
+
+    /**
+     * Validates Matriz.
+     */
+    @Test
+    public void countCharactersElephantSentence() {
+        final int size = 5;
+        final int ringNumber = 2;
+        Squarelotron squarelotron = new Squarelotron(size);
+        System.out.println(squarelotron.toString());
+        Squarelotron one = squarelotron.upsideDownFlip(ringNumber);
+        System.out.println(one.toString());
+        Squarelotron two = squarelotron.mainDiagonarlFlip(ringNumber);
+        System.out.println(two.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)